apps/meteor/client/views/room/body/hooks/useUnreadMessages.ts

```diff
@@ -73,8 +74,9 @@ export const useHandleUnread = (
 			return;
 		}
 		setMessageJumpQueryStringParameter(message?._id);
+		chat.readStateManager.markAsRead();
```

Comment:
This seemed to be fix for me. While clicking on the banner, It would take me to message. But when page is refreshed, the banner would reapper.